lib/game_turn.rb

```diff
@@ -0,0 +1,34 @@
+module TicTacToe
+  class GameTurn
+
+    def initialize(board, console_ui, eval_game, validation)
+      @board = board 
+      @console_ui = console_ui 
+      @eval_game = eval_game
+      @validation = validation
+    end
+
+    def execute(player, marker)
+      move = @console_ui.move_messages(@board.nine_space_array, player, marker)
+      move = valid_move_loop(move)
+      @board.fill(move, marker)
+      if @eval_game.won?(@board.nine_space_array)
+        @console_ui.display_board(@board.nine_space_array)
+        player
+      elsif @eval_game.tied?(@board.nine_space_array)
+        @console_ui.display_board(@board.nine_space_array)
+        0
+      end
+    end
```

Comment:
I don't think I would be expecting GameTurn#execute to return the game result. I would suggest removing the game result logic from this method and adding it into your EvalGame class since its responsibility is to evaluate the game. You can then evaluate the game result outside of this class